### PR TITLE
Fixed issue #387 broken mobile page 

### LINKED ISSUE #387: Mobile page Horizontally scrollable , Header overflows
This is probably the most visually significant bug .. **The mobile page is horizontally scrollable** , **which leads to overflowing of the header text** (i.e version and latest update) also in the end of every page , the mobile page is dragging upwards , **please see the Gif below for demostration** , I have also attached the **screenshots** from my **mobile phone** , please take a look.

Mobile Bug
photo_2022-04-07_20-03-19

### PR DESCRIPTION
**Pull request by Prakashh21**
This fixes #387 

This fix doesnot interfere with any other style in page as specified by @PintoGideon , 
**I have added Gif , of the results , take a look !**

ezgif com-gif-maker (5)

### CODE REVIEW

**Comment by Prakashh21** [comment]
@PintoGideon  In the latest commit I have , fixed version and release date (in the top right) flowing out of the page , take a look ! 

**Before**

Untitled drawing (8)

**After**

Screen Shot 2022-04-15 at 10 15 26

**Comment by PintoGideon** [comment]
The branch has conflicts. Could you resolve those @Prakashh21 ? The changes look good to me.

File: src/assets/scss/main.scss [hunk 1]
```diff
@@ -11,4 +11,32 @@
...
+}
+
+@media only screen and (max-width: $breakpoint-mobile){
+    .pf-c-page__header-brand{
+        padding-left:0px;
+        padding: 0.3rem;
+    }
+
+    .pf-c-page__header-brand-toggle{
+    margin: 0 !important;
```
Comment (jennydaman) [inline src/assets/scss/main.scss:35]: Please lint your code before submitting PR.

Why is `!important` necessary here?

Comment (Prakashh21) [reply src/assets/scss/main.scss:35]: ok I will keep that in mind , 
@jennydaman  I was not able to override the css without using , !important  at that time , tbh it was the last  straw , anyways I am looking into it ..

**Pull request closed without merging**